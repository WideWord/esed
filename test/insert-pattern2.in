awdawf
wefefwe
afefd
pwogfkwergpor
erkgpergk
regoprk
ad
gpoerk
