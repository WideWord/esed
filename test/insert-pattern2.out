awdawf
wefefwe
afefd
5678
pwogfkwergpor
erkgpergk
regoprk
ad
5678
gpoerk
